src/test/scala/firrtlTests/SerializerSpec.scala

```diff
@@ -58,4 +88,24 @@ class SerializerSpec extends AnyFlatSpec with Matchers {
     Serializer.serialize(stmts) should be(ser)
   }
 
+  it should "support emitting circuits" in {
+    val parsed = Parser.parse(simpleCircuit)
+    val serialized = Serializer.serialize(parsed)
+    serialized should be(simpleCircuit)
+  }
+
+  it should "support emitting individual modules" in {
+    val parsed = Parser.parse(simpleCircuit)
+    val m = parsed.modules.find(_.name == "test").get
+    val serialized = Serializer.serialize(m)
+    serialized should be(testModule)
+  }
+
+  it should "support emitting indented individual modules" in {
+    val parsed = Parser.parse(simpleCircuit)
+    val m = parsed.modules.find(_.name == "test").get
+    val serialized = Serializer.serialize(m, 1)
+    serialized should be(testModuleTabbed)
+  }
+
```

Comment:
The 3rd test is the formerly buggy behavior, the 1st 2 are to make sure I didn't break more typical emission in the process. These tests are whitespace exact.